entwatch/ze_fe8_sacred_stones_v1_10.cfg

```diff
@@ -116,8 +116,8 @@
 	}
 	"6"
 	{
-		"name"			"Zombie Skeleton"
-		"shortname"		"ZM Skeleton"
```

Comment:
You should leave ZM in the shortnames here at least, so they can be differentiated from human items on the hud.